docs/containers/bridge-to-kubernetes.md

```diff
@@ -198,4 +206,6 @@ Use the provided script to remove the sample application from your cluster.
 [supported-regions]: https://azure.microsoft.com/global-infrastructure/services/?products=kubernetes-service
```

Comment:
One of the links (server-js-breakpoint) still reference our old Azure Dev Spaces repo. Should we use microsoft/mindaro instead?